lib/ansible/modules/clustering/kubernetes.py

```diff
@@ -52,11 +52,13 @@
     required: false
     default: null
   patch_operation:
-    description:
-      - Specify patch operation for Kubernetes resource update. For details, see the description of PATCH operations at U(https://github.com/kubernetes/kubernetes/blob/release-1.5/docs/devel/api-conventions.md#patch-operations).
+    description: >
+      - Specify patch operation for Kubernetes resource update. For details, see the description of PATCH operations at
```

Comment:
@toidi That is the correct [folded scalar](http://www.yaml.org/spec/1.2/spec.html#style/block/folded) syntax.